pyramid_openapi3/exceptions.py

```diff
@@ -107,32 +107,28 @@ def extract_errors(
     ```
     """
     for err in errors:
-        if getattr(err, "schema_errors", None) is not None:
-            yield from extract_errors(request, err.schema_errors)
+        schema_errors = getattr(err, "schema_errors", None)
+        if schema_errors is not None:
```

Comment:
Should we use the walrus operator here? 

And bump minimum Python support to 3.8.

Reply:
We could do that. I am not sure how many people using this library are still on 3.7, but given the other breaking changes in this PR, it may not be a bad time to do it.

If we don't want to bump the minimum version, another option may be to do an `isinstance`

```python
if isinstance(err, InvalidSchemaValue):
    yield from extract_errors(request, err.schema_errors)
    continue
```

Import is `from openapi_core.unmarshalling.schemas.exceptions import InvalidSchemaValue`

or keep the 2 lines with the `getattr` as it is currently

Reply:
Let's keep `getattr` for now, to not introduce even more changes to this PR and drag it on for even longer.